code/index.html

```diff
@@ -4,27 +4,88 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
   </head>
-  <body>
+    <body>
+
     <!-- video or image as a header is cool :) -->
+      <div class="hero-image-container">
+        <div class="hero-title-container">
+          <h1><span>Pura Vida travels</span></h1>
+          <h2>Tailor-made trips to Costa Rica</h2>
+        </div>
+      </div>
+      
+    <!-- Wrap accordion and signup with flexbox -->
+
+  <div class="wrapper">
+
+    <!--Sign up -->
+    
+      <section class="contact-form">
+        <form action="https://httpbin.org/anything" method="POST">
+          <h1>Contact us</h1>
+
+          <div class="textfield">
+            <label>Full name</label>
+            <input type="text" name="" value="" placeholder="Write your name">
+          </div>
+
+          <div class="textfield">
+            <label>Email</label>
+            <input type="text" name="" value="" placeholder="Write your email">
+          </div>
+
+          <label class="checkbox">
+            <input type="checkbox" checked="checked">
+            <span class="checkmark">Get more info!</span>
+          </label>
+
+          <div class="button">
+            <input type="submit" name="Send">
+          
+        </form>
+
+      </section>
+
+    <!-- Accordion -->
+        <section class="accordion">
+          <div class="content">
+            <div class="question1">What packages do you offer?</div>
+            <div class="answer">
+              <p>Bla bla bla.</p>
+            </div>
+          </div>
+
+          <div class="content">
+            <div class="question2">What packages do you offer?</div>
+            <div class="answer">
+              <p>Bla bla bla.</p>
+            </div>
+          </div>
+
+          <div class="content">
+            <div class="question3">What packages do you offer?</div>
+            <div class="answer">
+              <p>Bla bla bla.</p>
+            </div>
+          </div>
+
+          <div class="content">
+            <div class="question4">What packages do you offer?</div>
+            <div class="answer">
+              <p>Bla bla bla.</p>
+            </div>
+          </div>
+
+        </section>
```

Comment:
Looks like you have a good start here! Just remember that class names are for CSS styling so maybe these can have the same class names but different ids (if they are going to have the same styling that is).